alert-common/src/main/java/com/synopsys/integration/alert/common/channel/message/ChannelMessageParser.java

```diff
@@ -158,19 +158,17 @@ protected String createLinkableItemString(LinkableItem linkableItem, boolean bol
         String value = encodeString(linkableItem.getValue());
         Optional<String> optionalUrl = linkableItem.getUrl();
 
-        String formattedString;
-        if (optionalUrl.isPresent()) {
-            String linkableItemValueString = createLinkableItemValueString(linkableItem);
-            formattedString = String.format("%s: %s", name, linkableItemValueString);
-        } else {
-            formattedString = String.format("%s: %s", name, value);
+        if (bold) {
+            name = emphasize(name);
+            value = emphasize(value);
         }
 
-        if (bold) {
-            formattedString = emphasize(formattedString);
+        if (optionalUrl.isPresent()) {
+            // The nuance around stylizing links adds too much complexity for too little value to worry about emphasizing the it.
```

Comment:
Is the end of the comment right? shouldn't it be "emphasizing it" not "emphasizing the it"?

Reply:
Haha, changed it from `value` to `it` because I didn't want to be redundant, but of course a forgot the `the`.